MPC/config/dds_openssl.mpb

```diff
@@ -1,4 +1,16 @@
 feature(openssl): openssl {
+  // Android preloads the system SSL library (either OpenSSL or BoringSSL) for
+  // the Java Android API, so we must:
+  //   1. Statically link OpenSSL to this library
+  //   2. Keep our OpenSSL symbols internal
+  // Number 1 is described in android.md. We are doing number 2 here.
+  verbatim (gnuace, top) {
+    comma := ,
+    exclude_openssl_libs := -Wl$(comma)--exclude-libs$(comma)libcrypto.a -Wl$(comma)--exclude-libs$(comma)libssl.a
```

Comment:
Does it really need `$(comma)`?  Would it be different if this was `=` instead of `:=`?

Reply:
> Does it really need $(comma)?

I copied this as is from #1066. I remember I spent quite a bit of time getting this working which led to the comma variable, I don't remember why it's needed though.

>  Would it be different if this was = instead of :=?

I try to prefer `:=` to `=`, but maybe.

Reply:
It's fine as-is, just wondering what was going on there.  Seems like it has to "hide" the comma from the `$(if`.

Reply:
Yeah that sounds familiar, so that's probably it.